test/models/UserAnswersSpec.scala

```diff
@@ -120,5 +121,59 @@ class UserAnswersSpec extends SpecBase {
         }
       }
     }
+
+    "removeDocumentFromItems" - {
```

Comment:
Just minor, I believe the test title should be removeTransportEquipmentFromItems